### PR TITLE
feat(NameGenerator): Add escher and hart

### PR DESCRIPTION
**Pull request by maxulysse**
Adding MC Esher and ViHart

### CODE REVIEW

**Comment by maxulysse** [comment]
cf https://github.com/nextflow-io/nextflow/issues/4910

**Comment by pditommaso** [comment]
The spell checker complains about this pr!

**Comment by maxulysse** [comment]
> The spell checker complains about this pr!

I know, which is why I opened this issue #4910

I'd want to add an ignore list and add `hart` to it

File: modules/nextflow/src/main/groovy/nextflow/util/NameGenerator.groovy [hunk 1]
```diff
@@ -397,6 +400,9 @@ class NameGenerator {
             // Margaret Hamilton - Director of the Software Engineering Division of the MIT Instrumentation Laboratory, which developed on-board flight software for the Apollo space program. https://en.wikipedia.org/wiki/Margaret_Hamilton_(scientist)
             "hamilton",
 
+            // Victoria "Vi" Hart - American mathematician and YouTuber. Recreational mathemusician, well-known for creating mathematical videos and popularizing mathematics. Hart founded the virtual reality research group eleVR and has co-authored several research papers on computational geometry and the mathematics of paper folding. https://en.wikipedia.org/wiki/Vi_Hart
+            "hart",
```
Comment (bentsherman) [inline modules/nextflow/src/main/groovy/nextflow/util/NameGenerator.groovy:404]: Not sure we should add someone who is so contemporary, that could open it up to many people and make it hard to define who should be included

Comment (maxulysse) [reply modules/nextflow/src/main/groovy/nextflow/util/NameGenerator.groovy:404]: I did add Pesquet already

Comment (pditommaso) [reply modules/nextflow/src/main/groovy/nextflow/util/NameGenerator.groovy:404]: 😂

**Comment by pditommaso** [comment]
Merged manually only adding Escher via 2e6496e28

**Pull request closed without merging**